### PR TITLE
Code of conduct

### PR DESCRIPTION
**Pull request by nijotz**
## Proposed Release Notes
* Fixed code of conduct links in contributing guide

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,9 +18,12 @@ licensing restrictions.
 
 ## Code of Conduct
 
-Before contributing please read the [code of conduct](./CODE_OF_CONDUCT.md)
+Before contributing please read the [code of
+conduct](https://github.com/newrelic/newrelic-repo-template/blob/main/CODE_OF_CONDUCT.md)
 
-Note that our [code of conduct](./CODE_OF_CONDUCT.md) applies to all platforms
+Note that our [code of
+conduct](https://github.com/newrelic/newrelic-repo-template/blob/main/CODE_OF_CONDUCT.md)
```
Comment (michaelgoin) [inline CONTRIBUTING.md:25]: Code of conduct link should be: https://github.com/newrelic/.github/blob/main/CODE_OF_CONDUCT.md

**Review by michaelgoin — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (nijotz) [reply CONTRIBUTING.md:25]: I'm trying to find where I even got that link so I could fix the source, and I can't even find it. I thought I got it from the oss-template or another one of our repos..🤷 oh well, fix coming soon, thanks for catching that

Comment (nijotz) [reply CONTRIBUTING.md]: pushed

**Review by carlo-808 — APPROVED** [review]
Nice 🎁  wrapping.  Links 👍

**Pull request merged**